src/libs/bin_decoder/lv_bin_decoder.c

```diff
@@ -362,12 +362,6 @@ void lv_bin_decoder_close(lv_image_decoder_t * decoder, lv_image_decoder_dsc_t *
     }
 
     free_decoder_data(dsc);
-
-    if(dsc->cache && dsc->cache_entry) {
```

Comment:
Can we make bin decoder behaves like other decoder, so we can call below code also in `lv_image_decoder_close`?

```c
    if(dsc->args.no_cache || !lv_image_cache_is_enabled()) decoder_draw_buf_free((lv_draw_buf_t *)dsc->decoded);

```

Reply:
On reflection, it doesn't make much sense to bring this code out into the public part. Since draw buffs are allocated within each individual module, it makes sense to release them internally. 😁